sim/warrior/thunder_clap.go

```diff
@@ -0,0 +1,56 @@
+package warrior
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (warrior *Warrior) RegisterThunderClapSpell() {
+	warrior.ThunderClapAuras = warrior.NewEnemyAuraArray(func(target *core.Unit) *core.Aura {
+		return core.ThunderClapAura(target)
+	})
+
+	warrior.ThunderClap = warrior.RegisterSpell(core.SpellConfig{
+		ActionID:       core.ActionID{SpellID: 6343},
+		SpellSchool:    core.SpellSchoolPhysical,
+		ProcMask:       core.ProcMaskRangedSpecial,
+		Flags:          core.SpellFlagIncludeTargetBonusDamage | core.SpellFlagAPL,
+		ClassSpellMask: SpellMaskThunderClap,
+
+		RageCost: core.RageCostOptions{
+			Cost: 20 - core.TernaryFloat64(warrior.HasMajorGlyph(proto.WarriorMajorGlyph_GlyphOfResonatingPower), 5, 0),
```

Comment:
Can this be moved to the static spell mods?

Reply:
Oh it's already in there, forgot to remove this